packages/workspace-tools-git/src/getDefaultRemote.ts

```diff
@@ -48,7 +47,7 @@ export function getDefaultRemote(cwdOrOptions: string | GetDefaultRemoteOptions)
 
   const gitRoot = findGitRoot(cwd);
 
-  let packageJson: Partial<PackageInfo> = {};
+  let packageJson: { repository?: string | { url?: string } } = {};
```

Comment:
Removed the PackageInfo type since that's now defined in a different package (which consumes this one)